src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerSession.cs

```diff
@@ -336,9 +347,13 @@ private Task StartAnalysis(IDependencyChainNode<PythonAnalyzerEntry> node, Async
             }
         }
 
-        private void LogCompleted(IPythonModule module, Stopwatch stopWatch, TimeSpan startTime) {
+        private void LogCompleted(IDependencyChainNode<PythonAnalyzerEntry> node, IPythonModule module, Stopwatch stopWatch, TimeSpan startTime) {
             if (_log != null) {
-                _log.Log(TraceEventType.Verbose, $"Analysis of {module.Name}({module.ModuleType}) completed in {(stopWatch.Elapsed - startTime).TotalMilliseconds} ms.");
+                var completed = node != null && module.Analysis is LibraryAnalysis ? "completed" : "completed for library";
```

Comment:
`var completed = node != null && module.Analysis is LibraryAnalysis ? "completed for library" : "completed";`

Reply:
This came from master...

Reply:
Thanks, fixed it in master :)